### DISCUSSION TITLE
Team creation in github classroom does not work

### DISCUSSION THREAD

**Discussion by rolandramp** [sebastian-hofstaetter/teaching#125] [Advanced IR: Exercise Q&A] [answered]
Hi,

I tried to create the Team for my group, but it produced following error:

GitHub team could not be created. There seems to be a problem on github.com, please try again.

Are there maybe some permissions missing?

Best 
Roland

**Comment by Kcooperative** [comment, marked as answer]
https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team

**Comment by pi-pa** [comment]
Is it still not working for you?
